### PR TITLE
{bio}[foss/2018a] CaVEMan v1.13.2

### PR DESCRIPTION
**Pull request by jdemeul**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/c/CaVEMan/CaVEMan-1.13.2-foss-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# The Francis Crick Insitute, London, UK
+##
+easyblock = 'CmdCp'
+
+name = 'CaVEMan'
+version = '1.13.2'
+
+homepage = 'http://cancerit.github.io/CaVEMan/'
+description = """SNV expectation maximisation based mutation calling algorithm 
+ aimed at detecting somatic mutaitons in paired (tumour/normal) cancer samples.
```
Comment (verdurin) [inline easybuild/easyconfigs/c/CaVEMan/CaVEMan-1.13.2-foss-2018a.eb]: @jdemeul Please correct the spelling of `mutations`...
Trivial point, I know.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.5.1804, Intel Xeon E312xx (Sandy Bridge), Python 2.7.5
See https://gist.github.com/d76bd27bfbb4ff7df7b1335792414478 for a full test report.

**Comment by jdemeul** [comment]
Thanks @verdurin for spotting that one 👍 
Would've forever been known as the cancer genomics researcher who couldn't spell "mutations" ...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3123.skitty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/fbe65344cd31c011485e48e49456a3ad for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2469.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d3f6fe850f48889b89adbd297ae59229 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2110.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ad083b7485ea85feca7d1ec9f997a83f for a full test report.

**Comment by boegel** [comment]
Going in, thanks @jdemeul!

**Pull request merged**